include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -346,6 +349,105 @@ def FormalOp : VerifOp<"formal", [
   }];
 }
 
+def SimulationOp : VerifOp<"simulation", [
+  DeclareOpInterfaceMethods<OpAsmOpInterface, ["getAsmBlockArgumentNames"]>,
+  IsolatedFromAbove,
+  NoTerminator,
```

Comment:
Why `NoTerminator` if we require a `verif.yield`? I'm actually a bit surprised that this didn't lead to any issues.

Reply:
Yikes, my bad. Pretty interesting that this didn't cause an issue anywhere :stuck_out_tongue: